README.md

```diff
@@ -7,15 +7,21 @@
 [![codecov.io](http://codecov.io/github/JuliaDiffEq/NeuralNetDiffEq.jl/coverage.svg?branch=master)](http://codecov.io/github/JuliaDiffEq/NeuralNetDiffEq.jl?branch=master)
 
 The repository is for the development of neural network solvers of differential equations such as physics-informed
-neural networks (PINNs) and deep BSDE solvers. It utilizes techniques like deep neural networks and neural 
+neural networks (PINNs) and deep BSDE solvers. It utilizes techniques like deep neural networks and neural
 stochastic differential equations to make it practical to solve high dimensional PDEs efficiently through the
 likes of scientific machine learning (SciML).
 
-# Examples
+## Installation
 
-## DeepBSDE Solver
+```bash
+julia ./install.jl
```

Comment:
This is very non-standard and we should stick to standard techniques, e.g. using the Pkg manager REPL (Also, this doesn't work on all operating systems)

Reply:
You can make a new environment for `examples` folder and add the dependencies to the Project.toml

Reply:
We should probably just have a tutorials section of the documentation like is done in DiffEqFlux, DataDrivenDiffEq, etc.

Reply:
Okay, working it.